pkgs/create-neon/test/create-neon.ts

```diff
@@ -1,17 +1,21 @@
 import { assert } from "chai";
 import { spawn } from "child_process";
-import execa from "execa";
 import * as path from "path";
 import { promises as fs } from "fs";
 import * as TOML from "toml";
 import expect from "../dev/expect";
 import rimraf from "../dev/rimraf";
 
+async function execa(command: string, args: readonly string[]): Promise<void> {
```

Comment:
`execa` is an ESM. Rather than adding complexity to building--because Node 14 doesn't provide a built-in loader--I chose to add a simple wrapper.

Reply:
Maybe just add a comment that we can remove this shim once we drop support for enough old versions of Node.